### DISCUSSION TITLE
%E5%A6%82%E4%BD%95%E5%9C%A8-github-%E4%B8%AD%E4%BD%BF%E7%94%A8-gpg-5b95d7654e4e441d9dde960ebba6122b

### DISCUSSION THREAD

**Discussion by giscus** [n1203/blog#5] [Show and tell]
# %E5%A6%82%E4%BD%95%E5%9C%A8-github-%E4%B8%AD%E4%BD%BF%E7%94%A8-gpg-5b95d7654e4e441d9dde960ebba6122b

1991年，程序员Phil Zimmermann为了避开政府监视，开发了加密软件PGP。这个软件非常好用，迅速流传开来，成了许多程序员的必备工具。但是，它是商业软件，不能自由使用。所以，自由软件基金会决定，开发一个PGP的替代品，取名为GnuPG。这就是GPG的由来。

http://localhost:3000/%E5%A6%82%E4%BD%95%E5%9C%A8-github-%E4%B8%AD%E4%BD%BF%E7%94%A8-gpg-5b95d7654e4e441d9dde960ebba6122b

**Comment by n1203** [comment]
👏 欢迎评论交流～
